composer.json

```diff
@@ -249,7 +249,7 @@
         "drupal/media_entity_image": "1.3",
         "drupal/media_entity_slideshow": "2.0-alpha1",
         "drupal/menu_block": "1.4.0",
-        "drupal/menu_breadcrumb": "^1.8",
+        "drupal/menu_breadcrumb": "^1.7",
```

Comment:
If we decide to rollback, the `^` will have to be removed from the version number otherwise it will just update to the new version.